### PR TITLE
 Add new window operation APIs to cover VD requirements.

### PR DESCRIPTION
**Pull request by an87li**
### Purpose of this PR

* Add new window operation APIs to cover VD requirements.

### Testing status

User get WindowAttributes in onCreate method

Tizen.Log.Fatal("OPENTK", "Set the window opacity to 0.1f");
WindowAttributes.WindowOpacity = 0.1f;

WindowAttributes.IsFocusAllowed = false;
Tizen.Log.Fatal("OPENTK", "Set the window is FocusAllowed as false");
Tizen.Log.Fatal("OPENTK", "Get the window is FocusAllowed is:" + WindowAttributes.IsFocusAllowed);

WindowAttributes.AddAuxiliaryHint("Test", "TestValue");

1. The window has no effect after set opacity
2. Set IsFocusAllowed to 'false', and then read it from low layer, the value also be 'false', so it works.
3. AddAuxiliaryHint 'Test' string, it is not a supported hint, SDL log will print 
I/SDL_LOG (10903): INFO: AddAuxiliaryHint: Not supported auxiliary hint [Test]
so it works.

### Comments
I use SDL method in app directly, it also no effect.

        [SuppressUnmanagedCodeSecurity]
        [DllImport("libSDL2-2.0.so.0", CallingConvention = CallingConvention.Cdecl, EntryPoint = "SDL_SetWindowOpacity", ExactSpelling = true)]
        public static extern int SDL_SetWindowOpacity(IntPtr window, float opacity);

 SDL_SetWindowOpacity(Window.WindowInfo.Handle, 0.0f);

### CODE REVIEW

**Comment by WonyoungChoi** [comment]
run Build Checker

File: src/OpenTK/Platform/Tizen/ITizenWindowAttributes.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        /// <summary>
+        /// Get or set whether the window accepts a focus or not.
+        /// </summary>
+        bool IsFocusAllowed { get; set; }
+
+        /// <summary>
+        /// Set opacity of the window
+        /// </summary>
+        float WindowOpacity { set; }
```
Comment (WonyoungChoi) [inline src/OpenTK/Platform/Tizen/ITizenWindowAttributes.cs]: The getter of WindowOpacity is needed also. 
You can use below SDL method.

/**
 *  \brief Get the opacity of a window.
 *
 *  If transparency isn't supported on this platform, opacity will be reported
 *  as 1.0f without error.
 *
 *  \param window The window in question.
 *  \param out_opacity Opacity (0.0f - transparent, 1.0f - opaque)
 *
 *  \return 0 on success, or -1 on error (invalid window, etc).
 *
 *  \sa SDL_SetWindowOpacity()
 */
extern DECLSPEC int SDLCALL SDL_GetWindowOpacity(SDL_Window * window, float * out_opacity);

**Review by WonyoungChoi — CHANGES_REQUESTED** [review]

File: src/OpenTK/Platform/Tizen/ITizenWindowAttributes.cs (hunk 1 above)
Comment (an87li) [reply src/OpenTK/Platform/Tizen/ITizenWindowAttributes.cs]: Added the getter ofr WindowOpacity, please review it again :)

**Review by WonyoungChoi — APPROVED** [review]

**Comment by WonyoungChoi** [comment]
LGTM, thanks

**Pull request merged**